internal/download/download.go

```diff
@@ -0,0 +1,29 @@
+package download
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/go-chi/chi"
+	"github.com/redhatinsights/insights-ingress-go/internal/config"
+	"github.com/redhatinsights/insights-ingress-go/internal/stage/filebased"
+)
+
+// NewHandler returns a http handler configured with a Pipeline
+func NewHandler(cfg config.IngressConfig) http.HandlerFunc {
+
+	return func(w http.ResponseWriter, r *http.Request) {
+		downloadID := chi.URLParam(r, "requestID")
+		fileName, filePath, err := filebased.GetFileStorageName(downloadID, cfg.StorageConfig.StorageFileSystemPath)
```

Comment:
Seems like `GetFileStorageName` should probably be a member function of the `filebased` Stager implementation?  :thinking:   Then we could stop passing in the `StorageFileSystemPath` and just use the `StagePath` member variable.  :thinking: 

We can make that change later.